recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "2.7.0" %}
+{% set version = "23.1" %}
```

Comment:
This is quite the jump. Is Pysal on CalVer now?

Reply:
Yes